src/test/java/org/jabref/gui/fieldeditors/LinkedFileViewModelTest.java

```diff
@@ -151,6 +182,31 @@ void deleteWhenDialogCancelledReturnsFalseAndDoesNotRemoveFile() {
         assertTrue(Files.exists(tempFile));
     }
 
+    // Structure taken from deleteWhenDialogCancelledReturnsFalseAndDoesNotRemoveFile method
+    @Test
+    void downloadHtmlFileCausesWarningDisplay() throws MalformedURLException {
+        Globals.prefs = JabRefPreferences.getInstance(); // required for task execution
```

Comment:
Please do not use JabRefPreferences.getInstance() as it operates on the real preferences, instead you can simply mock it as well: e,g, mock(JabRefPreferenes)
you can add this as field 
`    private final FilePreferences filePreferences = mock(FilePreferences.class);`

Reply:
Added mock of JabRefPreferences and reference in tests.